Makefile

```diff
@@ -258,6 +258,17 @@ gen_ci: .gitlab-ci.yml
 ci_clean:
 	rm -f .gitlab-ci.yml
 
+bender:
```

Comment:
Is this part of the `CI`-section of the makefile? or should we put this in a dedicated section?

Reply:
Currently this is only used in the CI, but I'm open to moving it to the main section and updating the `BENDER` variable on top. Updating the top variable would probably also be improved if the necessary make targets have the bender target as a prerequisite.